src/start.js

```diff
@@ -7,7 +7,7 @@ let started = false;
 
 export function start(opts) {
   started = true;
-  if (opts && opts.urlRerouteOnly) {
+  if (opts) {
```

Comment:
Is it possible that `!!opts === true && opts.urlRerouteOnly === undefined` ? Does that need to be checked?

Reply:
Yes, but it's okay if `urlRerouteOnly` is undefined, since it's just used for a truthiness check and will count as false. For a while, I was doing `opts.hasOwnProperty("urlRerouteOnly")`, but figured it wasn't worth doing that since there's no reason to provide options except to set urlRerouteOnly.

I guess it might be confusing though for `start()` to behave differently from `start({})`. I'll change it.